packages/gestalt/src/ListItem.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import * as React from 'react';
+import Box from './Box.js';
+import Text from './Text.js';
+import Touchable from './Touchable.js';
+
+type Props = {|
+  children: React.Node,
+  onClick: ({
+    event:
+      | SyntheticMouseEvent<HTMLDivElement>
+      | SyntheticKeyboardEvent<HTMLDivElement>,
+  }) => void,
+|};
+
+type State = {|
+  hovered: boolean,
+|};
+
+export default class ListItem extends React.PureComponent<Props, State> {
+  state = {
+    hovered: false,
+  };
+
+  handleMouseEnter = () => {
+    this.setState({ hovered: true });
+  };
+
+  handleMouseLeave = () => {
+    this.setState({ hovered: false });
+  };
+
+  render() {
+    const { children, onClick } = this.props;
+    const { hovered } = this.state;
+    return (
+      <Box role="listitem" padding={1} color={hovered ? 'lightGray' : 'white'}>
```

Comment:
@pfarejowicz Should this be hovered or focussed? Usually (e.g. Button) there should be no difference between hover & focus apart from the extra outline on focus.